### PR TITLE
load_new_products: Support loading MKM ids

### PR DESCRIPTION
**Pull request by kodawah**
I'm a bit unsure how to launch the script, and add it to autmation but beside that it should be good to add this chunk.

### CODE REVIEW

**Comment by kodawah** [comment]
all tested working, the addition of "MtG Set" may be controversial because I added some "custom" filtering as I didn't want to make the ignore file explode
tested by hardcoding the mkm secrets, I couldn't figure out how to run it from the cli (any help+documentation is welcome)

**Review by axxroytovu — COMMENTED** [review]
Content is good, need @ZeldaZach to review for integration with git automation.

File: data/review.yaml
Comment (axxroytovu) [inline data/review.yaml:1]: Don't include this file in the pull request, it gets generated automatically.

Comment (kodawah) [reply data/review.yaml:1]: > Don't include this file in the pull request, it gets generated automatically.

if it's the same I'd like to leave it in, in order to give git history an easier reference of what changes brought these in

**Review by ZeldaZach — APPROVED** [review]

**Pull request merged**